build/Nuget/Microsoft.NET.Nuget.proj

```diff
@@ -67,10 +39,6 @@
 
   <Target Name="Rebuild" DependsOnTargets="Clean;Build" />
 
-  <Target Name="LogOutput">
-    <Message Importance="High" Text="%(NuSpec.Identity) -> %(NuSpec.NuPkgOutput)'" />
-  </Target>
-
```

Comment:
Yes, add it back here (using different variables for lhs and rhs of `->` and remove the echo exec. In general, let's keep the diff to things that are related to the feature you want.